test/functional/disablewallet.py

```diff
@@ -19,11 +19,6 @@ def __init__(self):
         self.setup_clean_chain = True
         self.num_nodes = 1
 
```

Comment:
yes, I've put the extra_args back in. Good spot.

I don't understand why adding the "Keypool ran out" assert should be part of this PR?